perma_web/api/tests/test_current_user_resource.py

```diff
@@ -34,7 +31,7 @@ def test_get_archives_json(self):
         self.successful_get(self.detail_url + 'archives/', user=self.regular_user, count=11)
 
     def test_get_folders_json(self):
-        self.successful_get(self.detail_url + 'folders/', user=self.org_user, count=1)
+        self.successful_get(self.detail_url + 'folders/', user=self.org_user, count=2)
```

Comment:
See point 6 from the previous pull req (#1949): "/archives now includes all archives accessible_to the user, not those created_by the user. /folders now includes all top-level folders for the user, not all folders created_by the user. Rationale: this fixes a logic error in the v1 API. Since links and folders can be moved around and original creator can lose access to them, checking created_by made no sense."